### DISCUSSION TITLE
2023-07-07

### DISCUSSION THREAD

**Discussion by jaehee329** [woowacourse-teams/2023-haru-study#13] [daily_meeting]
### 오늘의 할 일
- 마코, 엽토
  - 발표 준비
- 룩소, 노아
  - 화면 디자인 및 필요한 데이터 명세
- 백엔드
  - URI 설계
- 회식 메뉴 정하기

### 다음주 월요일에 할 일

- 백엔드

  - 화요일까지 API 명세(모디 테오) 및 CI/CD (마코)

  - 다음 주 수요일까지는 MVP 내용 구현하기 - 미션 1차 제출 완성도 정도로

  - 기능 명세 이전에 OAuth 알아보기(주말동안)

- 프론트엔드

  - CI/CD(엽토)

  - 디자인(룩소, 노아)

- 전체
  
  - 월요일엔 프로필 관련 기능을 세부적으로 어떻게 제외할지를 논의한다.
  
  - 2차 데모까지 병렬적으로 준이 제안한 방법들을 기반으로 스터디를 따로 진행한다.

**Comment by jaehee329** [comment]
9점 - 괜찮게 자서 낫배드한 컨디션. 드디어 주말이네요~

9점 - 일단 주말이어서 좋다~ 즐겁게 회식하고 쉬러 가죠 우리

**Comment by nlom0218** [comment]
### 체크인

9점 - 벌써 금요일이라니~!~!~! 시간이 참 빨라서 -1

### 체크아웃

6점 - 청년 수당 떨어져서 -2, 생각할 것들이 많아져서 -2

**Comment by woo-jk** [comment]
### 체크인
9점 - 잠 늦게자서 -2, 오늘 지하철 안타서 +1

### 체크아웃
10점 - 팀 회식 하러가서 기분이 좋다 ㅎㅎ 우리 서비스의 차별성이 약간 약하다고 생각하고 있었는데 준이 그 응어리를 많이 풀어줘서 좋았다.

**Comment by woosung1223** [comment]
### 체크인
8점 - 일어나는데 힘들었읍니다

### 체크아웃
10점 - 데모데이 다들 너무 고생했습니다! 👍 
          기능이 변경되어야 하는 부분은 있고 논의가 길어지기야 하겠지만 좋은 흐름이라고 생각됩니다!!

**Comment by yeopto** [comment]
### 체크인

10점 - 늦잠 잤는데 지각 안해서 기분이 좋다

### 체크아웃

3점 - 회의 중 모디에게 격양되어 직설적으로 말했는데 미안해서 -5점, 데모데이 발표 후 고민거리가 더 많아져서 -2점

**Comment by MoonJeWoong** [comment]
### 체크인
10점 - 오늘 지나면 회식도 있고, 한 주가 잘 정리되어가는 기분이라 10점입니다.

### 체크아웃
9점 - 곰곰히 다시 생각해보니 그렇게 또 많이 바뀔게 있을까 싶어서 좀 괜찮아 진 것 같습니다!

**Comment by aak2075** [comment]
### 체크인

10점 - 적당히 자서 컨디션도 좋고 1차 스프린트 끝나는 날이라 좋다. 발표 화이팅💪

### 체크아웃

10점 - 발표 마무리 돼서 후련하다. 시원하게 한 잔 합시다🍻
